cli/src/main/java/com/box/l10n/mojito/cli/command/PseudoLocCommand.java

```diff
@@ -0,0 +1,148 @@
+
+@Component
+@Scope("prototype")
+@Parameters(commandNames = {"pseudo", "pl"}, commandDescription = "Pull pseudo localized assets from TMS")
+public class PseudoLocCommand extends Command {
+
+    /**
+     * logger
+     */
+    static Logger logger = LoggerFactory.getLogger(PseudoLocCommand.class);
+
+    @Autowired
+    ConsoleWriter consoleWriter;
+
+    @Parameter(names = {Param.REPOSITORY_LONG, Param.REPOSITORY_SHORT}, arity = 1, required = true, description = Param.REPOSITORY_DESCRIPTION)
+    String repositoryParam;
+
+    @Parameter(names = {Param.SOURCE_DIRECTORY_LONG, Param.SOURCE_DIRECTORY_SHORT}, arity = 1, required = false, description = Param.SOURCE_DIRECTORY_DESCRIPTION)
+    String sourceDirectoryParam;
+
+    @Parameter(names = {Param.TARGET_DIRECTORY_LONG, Param.TARGET_DIRECTORY_SHORT}, arity = 1, required = false, description = Param.TARGET_DIRECTORY_DESCRIPTION)
+    String targetDirectoryParam;
+
+    @Parameter(names = {Param.FILE_TYPE_LONG, Param.FILE_TYPE_SHORT}, arity = 1, required = false, description = Param.FILE_TYPE_DESCRIPTION,
+            converter = FileTypeConverter.class)
+    FileType fileType;
+
+    @Parameter(names = {Param.SOURCE_LOCALE_LONG, Param.SOURCE_LOCALE_SHORT}, arity = 1, required = false, description = Param.SOURCE_LOCALE_DESCRIPTION)
+    String sourceLocale;
+
+    @Parameter(names = {Param.SOURCE_REGEX_LONG, Param.SOURCE_REGEX_SHORT}, arity = 1, required = false, description = Param.SOURCE_REGEX_DESCRIPTION)
+    String sourcePathFilterRegex;
+
+    @Autowired
+    AssetClient assetClient;
+
+    @Autowired
+    CommandHelper commandHelper;
+
+    Repository repository;
+
+    CommandDirectories commandDirectories;
+
+    /**
+     * Contains a map of locale for generating localized file a locales defined
+     * in the repository.
+     */
+    Map<String, String> localeMappings;
+
+    @Override
+    public void execute() throws CommandException {
+        consoleWriter.newLine().a("Pull pseudo localized asset from repository: ").fg(Color.CYAN).a(repositoryParam).println(2);
+
+        repository = commandHelper.findRepositoryByName(repositoryParam);
+
+        commandDirectories = new CommandDirectories(sourceDirectoryParam, targetDirectoryParam);
+
+        for (FileMatch sourceFileMatch : commandHelper.getSourceFileMatches(commandDirectories, fileType, sourceLocale, sourcePathFilterRegex)) {
+            consoleWriter.a("Localizing: ").fg(Color.CYAN).a(sourceFileMatch.getSourcePath()).println();
+            generateLocalizedFiles(repository, sourceFileMatch);
+        }
+        consoleWriter.fg(Color.GREEN).newLine().a("Finished").println(2);
+    }
+
+    /**
+     * Default generation, uses the locales defined in the repository to
+     * generate the localized files.
+     *
+     * @param repository
+     * @param sourceFileMatch
+     * @throws CommandException
+     */
+    void generateLocalizedFiles(Repository repository, FileMatch sourceFileMatch) throws CommandException {
```

Comment:
- update the documentation
- rename the function: this is not generating locaclized file"s" anymore but just the pseudo localized one